packages/flutter/lib/src/material/slider.dart

```diff
@@ -535,6 +541,30 @@ class _RenderSlider extends RenderBox {
 
   bool get isDiscrete => divisions != null && divisions > 0;
 
+  @override
+  void attach(PipelineOwner owner) {
+    super.attach(owner);
+    _reaction = new CurvedAnimation(
+      parent: _state.reactionController,
+      curve: Curves.fastOutSlowIn,
+    )..addListener(markNeedsPaint);
+    _state.enableController.value = isInteractive ? 1.0 : 0.0;
+    _enableAnimation = new CurvedAnimation(
+      parent: _state.enableController,
+      curve: Curves.easeInOut,
+    )..addListener(markNeedsPaint);
+    _state.positionController.value = _value;
+    _state.positionController.addListener(markNeedsPaint);
+  }
+
+  @override
+  void detach() {
+    super.detach();
```

Comment:
by convention the super call in detach comes last, to be symmetric with attach

Reply:
Done